tools/lint-js.js

```diff
@@ -86,9 +86,7 @@ if (cluster.isMaster) {
     outFn = function(str) {
       fs.writeSync(fd, str, 'utf8');
     };
-    process.on('exit', function() {
-      fs.closeSync(fd);
-    });
+    process.on('exit', () => { fs.closeSync(fd); });
```

Comment:
```suggestion
    process.on('exit', () => fs.closeSync(fd));
```

Reply:
I prefer it with `{` and `}` because it makes it clear that the function is not returning a value.